src/components/admin/settings/general/Update.vue

```diff
@@ -1,79 +1,5 @@
-<script setup lang="ts">
-import { generalSettingsSchema } from '~/schemas/setting';
-import type { z } from "zod";
-
-defineProps<{
-  forms?: 'general' | 'seo' | 'all';
-}>();
-
-const emits = defineEmits<{
-  saved: [];
-}>();
-
-
-const { url, tick } = useRemoteAsset('orgImage');
-
-const generalSettings = useGeneralSettings();
-
-// Define form schema and use it in the form handling
-const formSchema = toTypedSchema(generalSettingsSchema);
-const { handleSubmit, errors, defineField } = useForm({
-  validationSchema: formSchema,
-});
-
-// Organization Fields
-const [organizationName] = defineField('organization.name');
-const [organizationDescription] = defineField('organization.description');
-const [organizationLocation] = defineField('organization.location');
-const [organizationLinks] = defineField('organization.links');
-
-// SEO Fields
-const [seoTitle] = defineField('seo.title');
-const [seoDescription] = defineField('seo.description');
-const [seoTwitter] = defineField('seo.twitter');
-
-const {
-  remove: removeFeaturedLink,
-  push: addFeaturedLink,
-  fields: featuredLinks,
-} = useFieldArray<z.infer<typeof generalSettingsSchema>['organization']['links'][0]>('organization.links');
-
-// Initialize fields with data from settings
-watchEffect(() => {
-  if (generalSettings.data.value) {
-    const gs = generalSettings.data.value;
-
-    organizationName.value = gs.organization.name;
-    organizationDescription.value = gs.organization.description;
-    organizationLocation.value = gs.organization.location;
-    organizationLinks.value = gs.organization.links;
-
-    seoTitle.value = gs.seo.title;
-    seoDescription.value = gs.seo.description;
-    seoTwitter.value = gs.seo.twitter;
-  }
-});
-
-const isSubmitting = ref(false);
-const onSubmit = handleSubmit(async values => {
-  try {
-    isSubmitting.value = true;
-    await $fetch('/api/settings/general', {
-      method: 'PUT',
-      body: { ...values },
-    });
-
-    emits('saved');
```

Comment:
you are changing unnecessary things, review your changes first. This is coming from not rebasing the branch.